ide/ide.kit/test/qa-functional/src/org/netbeans/test/ide/CountingSecurityManager.java

```diff
@@ -112,7 +112,7 @@ public static void initialize(String prefix, Mode mode, Set<String> allowedFiles
 
     static void assertReflection(int maxCount, String whitelist) {
         System.setProperty("counting.reflection.whitelist", whitelist);
-        System.getSecurityManager().checkMemberAccess(null, maxCount);
+        //System.getSecurityManager().checkMemberAccess(null, maxCount);
```

Comment:
You can replace this call with a [trick like this](https://github.com/apache/incubator-netbeans/pull/790/commits/8644c7993568081a62104c539441de1f07cb4f9c), I think.